pages/linux/systemd-resolve.md

```diff
@@ -0,0 +1,8 @@
+# systemd-resolve
+
+> This command is an alias of `resolvectl`.
+> More information: <https://manpages.ubuntu.com/manpages/xenial/en/man1/systemd-resolve.1.html>.
+
+- View documentation for the original command:
+
+`tldr resolvectl`
```

Comment:
How about this?
```suggestion
# systemd-resolve

> Resolve domain names, IPV4 and IPv6 addresses, DNS resource records, and services.
> Note: this tool has been renamed to `resolvectl` in new versions of systemd.
> More information: <https://manned.org/systemd-resolve>.

- View documentation for `resolvectl`:

`tldr resolvectl`
```

Reply:
Yeah, this LGTM.

Reply:
> How long do we keep waiting for @quantumflo?

Ideally, we wait for 1 week for the author to respond (before applying the changes ourselves) and we wait for 1 month in case of larger PRs, after which we can take over the PR and perform any additional changes.